ui/page/claim/internal/claimPageComponent/internal/channelPage/tabs/contentTab/internal/searchResults.jsx

```diff
@@ -32,18 +42,19 @@ export function SearchResults(props: Props) {
   }, [searchResults, onResults]);
 
   React.useEffect(() => {
+    if (noMoreResults.current) return;
+    isSearching.current = true;
     const timer = setTimeout(() => {
       if (searchQuery.trim().length < 3 || !claimId) {
         return setSearchResults(null);
       }
 
-      setIsSearching(true);
-
       lighthouse
         .search(
           `from=${SEARCH_PAGE_SIZE * (page - 1)}` +
-            `&s=${encodeURIComponent(searchQuery)}` +
+            `&s=${'%22' + encodeURIComponent(searchQuery) + '%22'}` +
             `&channel_id=${encodeURIComponent(claimId)}` +
```

Comment:
Hm ... searching "google" in Sam's page doesn't work with the extra quotes.  It's double double-quoting, compared to the Search Page's.

<img width="122" alt="image" src="https://github.com/OdyseeTeam/odysee-frontend/assets/64950861/f3c79851-dbee-4a49-9099-bdbac3c0c575">

Reply:
Without the forced one, `google react` produces something in Sam's page.  With forced, it produces nothing.